horreum-backend/src/main/resources/db/changeLog.xml

```diff
@@ -2360,5 +2360,65 @@
             CREATE POLICY dataset_update ON dataset FOR UPDATE USING (has_role2(owner, 'tester'));
         </sql>
     </changeSet>
-</databaseChangeLog>
 
+    <changeSet id="58" author="jwhiting">
+       <createSequence sequenceName="label_id_seq" startValue="1" incrementBy="1" cacheSize="1" />
+       <createTable tableName="label">
+           <column name="id" type="bigint">
+               <constraints primaryKey="true" nullable="false" />
+           </column>
+           <column name="label" type="text">
+               <constraints nullable="false" />
+           </column>
+           <column name="value" type="text">
+               <constraints nullable="false" />
+           </column>
+           <column name="datasetid" type="integer" />
+           <column name="filtering" type="boolean" defaultValue="true"/>
+           <column name="metrics" type="boolean" defaultValue="true"/>
+       </createTable>
+       <sql>
+            GRANT SELECT, INSERT, DELETE, UPDATE ON TABLE label TO "${quarkus.datasource.username}";
+            ALTER TABLE label ENABLE ROW LEVEL SECURITY;
+            CREATE POLICY l_select ON label FOR SELECT
+                USING (exists(SELECT 1 FROM test WHERE test.id = test_id AND can_view2(test.access, test.owner)) OR has_role('horreum.system'));
```

Comment:
There's no `test_id` in `label` and there shouldn't be, Labels are a component of Schema, though with it's own access control.